pkg/util/azureclient/mgmt/redhatopenshift/openshiftclusters.go

```diff
@@ -12,7 +12,7 @@ import (
 
 	"github.com/Azure/go-autorest/autorest"
 
-	"github.com/Azure/ARO-RP/pkg/client/services/redhatopenshift/mgmt/2020-04-30/redhatopenshift"
+	"github.com/Azure/ARO-RP/pkg/client/services/redhatopenshift/mgmt/2020-12-31-preview/redhatopenshift"
```

Comment:
Do we want to switch to preview at this point?

Reply:
No, not yet.

Reply:
@petrkotas  can we rollback this for time being, not switch the versions yet